src/app/components/event-card/event-card.component.html

```diff
@@ -1,25 +1,34 @@
-<mat-card class="mat-card">
+<mat-card class="mat-card" (click)="this.viewEmitter.emit()">
   <mat-card-header class="mat-card-header">
-    <mat-card-title class="mat-card-title">{{ event.name }}</mat-card-title>
-    <div>
-      <mat-icon (click)="this.viewEmitter.emit()" class="icon-hover"
+    <mat-card-title class="mat-card-title">{{ event?.name }}</mat-card-title>
+    <div *ngIf="showButtons">
+      <mat-icon
+        (click)="this.viewEmitter.emit()"
+        class="icon-hover"
+        appClickStopPropagation
         >open_in_new</mat-icon
       >
-      <mat-icon (click)="this.deleteEmitter.emit()" class="icon-hover"
+      <mat-icon
+        (click)="this.editEmitter.emit()"
+        class="icon-hover"
+        appClickStopPropagation
         >edit</mat-icon
       >
-      <mat-icon (click)="this.editEmitter.emit()" class="icon-hover"
+      <mat-icon
+        (click)="this.deleteEmitter.emit()"
+        class="icon-hover"
+        appClickStopPropagation
         >delete</mat-icon
       >
     </div>
   </mat-card-header>
 
   <div class="tags-container">
     <app-tag>Event</app-tag>
```

Comment:
What is tag component used for? Why not just use <p> and <div> to display the event tags and 'Event' text?